blocks/init/src/Blocks/components/embed/components/embed-options.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { ToggleControl, TextControl, Notice } from '@wordpress/components';
+import {
+	UseToggle,
+	checkAttr,
+	getAttrKey
+} from '@eightshift/frontend-libs/scripts';
+import manifest from '../manifest.json';
+
+export const EmbedOptions = (attributes) => {
+	const {
+		title: manifestTitle,
+	} = manifest;
+
+	const {
+		setAttributes,
+		label = manifestTitle,
+		embedShowControls = true,
+
+		showEmbedUse = true,
+		showEmbedUrl = true,
+		showEmbedTitle = true,
+		showEmbedAdvanced = true,
+		showEmbedAutoplay = true,
+		showEmbedEncryptedMedia = true,
+		showEmbedPictureInPicture = true,
+		showEmbedAllowFullScreen = true,
+	} = attributes;
+
+	if (!embedShowControls) {
+		return null;
+	}
+
+	const embedUse = checkAttr('embedUse', attributes, manifest);
+	const embedUrl = checkAttr('embedUrl', attributes, manifest);
+	const embedTitle = checkAttr('embedTitle', attributes, manifest);
+	const embedAutoplay = checkAttr('embedAutoplay', attributes, manifest);
+	const embedEncryptedMedia = checkAttr('embedEncryptedMedia', attributes, manifest);
+	const embedPictureInPicture = checkAttr('embedPictureInPicture', attributes, manifest);
+	const embedAllowFullScreen = checkAttr('embedAllowFullScreen', attributes, manifest);
+
+	const [showAdvanced, setShowAdvanced] = useState(false);
+
+	return (
+		<UseToggle
+			label={sprintf(__('%s', 'safer-internet'), label)}
```

Comment:
Please never do this
```suggestion
			label={label}
```